core/src/main/java/io/seata/core/rpc/netty/AbstractRpcRemotingClient.java

```diff
@@ -113,19 +113,23 @@ public void run() {
                 clientChannelManager.reconnect(getTransactionServiceGroup());
             }
         }, SCHEDULE_INTERVAL_MILLS, SCHEDULE_INTERVAL_MILLS, TimeUnit.SECONDS);
-        mergeSendExecutorService = new ThreadPoolExecutor(MAX_MERGE_SEND_THREAD,
-            MAX_MERGE_SEND_THREAD,
-            KEEP_ALIVE_TIME, TimeUnit.MILLISECONDS,
-            new LinkedBlockingQueue<>(),
-            new NamedThreadFactory(getThreadPrefix(), MAX_MERGE_SEND_THREAD));
-        mergeSendExecutorService.submit(new MergedSendRunnable());
+        if (NettyClientConfig.isEnableClientBatchSendRequest()) {
+            mergeSendExecutorService = new ThreadPoolExecutor(MAX_MERGE_SEND_THREAD,
```

Comment:
I m wondering why close the mergeSendExecutorService? If disable "isEnableClientBatchSendRequest", the baseketMap always empty.

Reply:
> I m wondering why close the mergeSendExecutorService? If disable "isEnableClientBatchSendRequest", the baseketMap always empty.

If it is a single send, you do not need mergeSendExecutorService